flexvol/flexvoldriver.go

```diff
@@ -275,17 +278,20 @@ func mount(dir, opts string) error {
 
 	ninputs, workloadPath, s := checkValidMountOpts(opts)
 	if !s {
-		return failure("mount", inp, "Incomplete inputs")
+		logError("mount", inp, "Incomplete inputs", syslogOnlyTrue)
```

Comment:
shouldn't this be `sysLogOnlyFalse` --- i.e. in the original code it would have logged to stdout

Reply:
I changed some of the existing behavior to prevent flooding due to periodic or retry-type loops. I _think_ that includes all the code in mount(), unbount() calls. Wdyt?

Reply:
I don't think so.  `mount()` is the top level command that gets invoked from the command line library, so when it fails we need to return a JSON response with the error.

Reply:
Same with `unmount()`

Reply:
Fixed